pulsar-broker/src/main/java/org/apache/pulsar/broker/service/Topic.java

```diff
@@ -110,7 +110,7 @@ default long getOriginalSequenceId() {
 
     CompletableFuture<Void> checkReplication();
 
-    CompletableFuture<Void> close();
+    CompletableFuture<Void> closeTopic();
```

Comment:
why do you change the name?

Reply:
`TransactionBuffer` has the same method.

Reply:
maybe we should just change TransactionBuffer ?

Reply:
But the TransactionBuffer interface is extended `AutoClosable`. So I can't change it.

Reply:
Since TransactionBuffer is new code, let's change TransactionBuffer rather than changing the existing code. Probably we can just get rid of `extend AutoClosable` and add `CompletableFuture<Void> close()` to `TransactionBuffer` interface.

Reply:
Oh. I see. I will replace it.